test/library/standard/GMP/studies/gmp-chudnovsky.chpl

```diff
@@ -312,7 +312,7 @@ proc my_sqrt_ui(ref r: mpf_t, x: c_ulong) {
   }
 
   mpf_set_prec_raw(t1, DOUBLE_PREC);
-  mpf_set_d(t1, 1/sqrt(x));
+  mpf_set_d(t1, 1.0/sqrt(x:real));
```

Comment:
Are these casts still necessary?  I thought you were striving to preserve this behavior?  (though I could be misremembering some consequence).

Reply:
No they aren't necessary anymore. I ended up leaving them in for some tests but I could remove them if you like.

Reply:
I think that'd be good to do, just to minimize the diff and avoid implying that changes were required when they weren't.

Reply:
OK, I've removed this change as well as the similar changes to studies/madness.